README.md

```diff
@@ -29,14 +38,18 @@ The example uses the [World Cities Dataset](https://simplemaps.com/data/world-ci
   }
 ```
 
-### Database Setup
+## Database setup
 
 It's recommended to use [`Postgres`] for your database so the schema can use `BigInt` and `Float` nicely.
```

Comment:
[dust] We seem to format it as `Postgres` in some instances and just Postgres in others. Not a big deal, but consistency one way or the other would be good.